Update Html.php

dubox opened the pull request:
Add position attr for image whitch driving by MemoryDrawing

This is:

- [x] a bugfix
- [ ] a new feature
- [ ] refactoring
- [ ] additional unit tests

Checklist:

- [ ] Changes are covered by unit tests
  - [ ] Changes are covered by existing unit tests
  - [x] New unit tests have been added
- [ ] Code style is respected
- [ ] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [ ] CHANGELOG.md contains a short summary of the change and a link to the pull request if applicable
- [ ] Documentation is updated as necessary

### Why this change is needed?

[https://github.com/PHPOffice/PhpSpreadsheet/pull/3529](https://github.com/PHPOffice/PhpSpreadsheet/pull/3529)

oleibman commented:
I have updated your description to point to issue 3529 rather than 3530. If you feel it should be 3530, please explain why.

You need to include a unit test to demonstrate that your code is working as desired. It might also be helpful to include a sample so that we can see the practical effect of your change on the output file.

dubox commented:
> I have updated your description to point to issue 3529 rather than 3530. If you feel it should be 3530, please explain why.
> 
> You need to include a unit test to demonstrate that your code is working as desired. It might also be helpful to include a sample so that we can see the practical effect of your change on the output file.

I have added a unittest at tests/PhpSpreadsheetTests/Writer/Html/MemoryDrawingOffsetTest.php

oleibman commented:
Thank you for supplying the test. I have a few minor changes that I would like you to make to it, but won't be able to document those until tonight.

oleibman commented on tests/PhpSpreadsheetTests/Writer/Html/MemoryDrawingOffsetTest.php:
```diff
@@ -0,0 +1,50 @@
+        $sheet = $spreadsheet->getActiveSheet();
+        $drawing = new MemoryDrawing();
+        if (($image = file_get_contents('https://avatars.githubusercontent.com/u/1836015')) === false) {
+            throw new Exception('image err1');
```
Having to fetch the drawing from the web adds an extra dependency to this test, one that is certainly not needed for it. Please save the drawing as a local file, somewhere like `tests/data/Reader/HTML/memoryDrawingTest.jpg`. After that, it is not necessary to throw an exception when file_get_contents or imagecreatefromstring fails - you should just assert that the result of the function call is not false. Your test code should look something like:
```php
        $image = file_get_contents('tests/data/Reader/HTML/memoryDrawingTest.jpg');
        self::assertNotFalse($image, 'unable to read file');
        $image = imagecreatefromstring($image);
        self::assertNotFalse($image, 'unable to create image from string');
```

dubox replied:
I have done those, and thank u for point out the problems in my code.
